### ISSUE TITLE
Synesthesia

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10247]
**Preferred term label:**
Synesthesia

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0233778

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Salvia divinorum toxicity

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000024
- label: Synesthesia
- parents: HP:0011446
- pmid: PMID:32114456
- def: A phenomenon whereby stimulation of one sensory or cognitive pathway leads to automatic, involuntary experiences in a second sensory or cognitive pathway. For instance, people with synaesthesia may experience unusual colours, tastes, and other sensations when engaged in everyday activities like reading or listening to music.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10247
- synonyms:
